src/components/Todo/Todo.tsx

```diff
@@ -18,6 +24,11 @@ export const Todo: React.FC<TodoProps> = ({ id, title, descr, deleteTodo }) => {
           btnClass="round-btn round-btn--red"
           onClickBtn={() => deleteTodo(id)}
         />
+        <RoundButton
+          btnText="E"
+          btnClass="round-btn round-btn--yellow"
+          onClickBtn={() => handleEditVisibleModal(id)}
```

Comment:
Does this function edit a visible modal?

Reply:
changed